app/controllers/candidate_interface/offer_dashboard_controller.rb

```diff
@@ -10,6 +10,7 @@ def show
       @application_choice = choices.pending_conditions.first || choices.recruited.first || choices.offer_deferred.first
       @provider_name = @application_choice.current_provider.name
       @course_name_and_code = @application_choice.current_course.name_and_code
+      @provider = @application_choice.current_provider
```

Comment:
Should we then remove line 11 and replace `@provider_name` with `@provider.name`?

Reply:
Or similar to what I suggested in my previous comment, using `@application_choice` in the component.